### PR TITLE
Issue #6949: spell checker is failing due to failure to get words file

### PR DESCRIPTION
**Pull request by romani**
Issue #6949

testing ....

### CODE REVIEW

**Comment by romani** [comment]
Looks like dictionaries are installed by some package....

**Comment by romani** [comment]
@pbludov , please review.

File: .ci/jsoref-spellchecker/whitelist.words [hunk 1]
```diff
@@ -365,6 +365,7 @@ edb
 edef
 edf
 edu
+Eghj
```
Comment (rnveach) [inline .ci/jsoref-spellchecker/whitelist.words]: I am not seeing where this word came from in PR.

Comment (romani) [reply .ci/jsoref-spellchecker/whitelist.words]: It is a leack from your commit on new arguments

Comment (rnveach) [reply .ci/jsoref-spellchecker/whitelist.words]: I see it now, thanks.

**Comment by romani** [comment]
@pbludov , I merging this quicker to activate spellcheker in CI, but please review a change to be familiar with spellcheker a bit.

**Pull request merged**